python/dstat-kudu/kudu_dstat.py

```diff
@@ -4,16 +4,12 @@
 import sys
 import tempfile
 import time
+from kudu.client import Partitioning
```

Comment:
Kinda gave this a shot by including the diff file in my testbed, it looks like it couldn't recognize the Partitioning symbol. Perhaps our bindings under examples is not in sync with the main repo python client ?